monitoring/nagios.py

```diff
@@ -51,6 +51,11 @@
        Only usable with the C(downtime) action.
     required: false
     default: Ansible
+  comment:
```

Comment:
I would default it to null (None/empty) as that follows current behavior, adding it with a default changes this